tests/Oryx.Integration.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -129,7 +129,7 @@ public async Task CanBuildAndRunPythonApp_UsingScriptCommandAndSetEnvSwitch()
                 .SetEnvironmentVariable(
                     SdkStorageConstants.SdkStorageBaseUrlKeyName,
                     SdkStorageConstants.DevSdkStorageBaseUrl)
-                .AddCommand($"oryx create-script -appPath {appDir} -bindPort {ContainerPort}")
+                .AddCommand($"oryx create-script -appPath {appDir} -bindPort {ContainerPort} -enableDynamicInstall")
```

Comment:
is that the fix to current validation build failure bug in master?

Reply:
This is the PR having the fix. https://github.com/microsoft/Oryx/pull/505